extensions/mssql/src/mssql.d.ts

```diff
@@ -153,85 +153,112 @@ declare module 'mssql' {
-		ignoreSemicolonBetweenStatements: boolean;
-		ignoreRouteLifetime: boolean;
-		ignoreRoleMembership: boolean;
-		ignoreQuotedIdentifiers: boolean;
-		ignorePermissions: boolean;
-		ignorePartitionSchemes: boolean;
-		ignoreObjectPlacementOnPartitionScheme: boolean;
-		ignoreNotForReplication: boolean;
-		ignoreLoginSids: boolean;
-		ignoreLockHintsOnIndexes: boolean;
-		ignoreKeywordCasing: boolean;
-		ignoreIndexPadding: boolean;
-		ignoreIndexOptions: boolean;
-		ignoreIncrement: boolean;
-		ignoreIdentitySeed: boolean;
-		ignoreUserSettingsObjects: boolean;
-		ignoreFullTextCatalogFilePath: boolean;
-		ignoreWhitespace: boolean;
-		ignoreWithNocheckOnForeignKeys: boolean;
-		verifyCollationCompatibility: boolean;
-		unmodifiableObjectWarnings: boolean;
-		treatVerificationErrorsAsWarnings: boolean;
-		scriptRefreshModule: boolean;
-		scriptNewConstraintValidation: boolean;
-		scriptFileSize: boolean;
-		scriptDeployStateChecks: boolean;
-		scriptDatabaseOptions: boolean;
-		scriptDatabaseCompatibility: boolean;
-		scriptDatabaseCollation: boolean;
-		runDeploymentPlanExecutors: boolean;
-		registerDataTierApplication: boolean;
-		populateFilesOnFileGroups: boolean;
-		noAlterStatementsToChangeClrTypes: boolean;
-		includeTransactionalScripts: boolean;
-		includeCompositeObjects: boolean;
-		allowUnsafeRowLevelSecurityDataMovement: boolean;
-		ignoreWithNocheckOnCheckConstraints: boolean;
-		ignoreFillFactor: boolean;
-		ignoreFileSize: boolean;
-		ignoreFilegroupPlacement: boolean;
-		doNotAlterReplicatedObjects: boolean;
-		doNotAlterChangeDataCaptureObjects: boolean;
-		disableAndReenableDdlTriggers: boolean;
-		deployDatabaseInSingleUserMode: boolean;
-		createNewDatabase: boolean;
-		compareUsingTargetCollation: boolean;
-		commentOutSetVarDeclarations: boolean;
-		blockWhenDriftDetected: boolean;
-		blockOnPossibleDataLoss: boolean;
-		backupDatabaseBeforeChanges: boolean;
-		allowIncompatiblePlatform: boolean;
-		allowDropBlockingAssemblies: boolean;
-		dropConstraintsNotInSource: boolean;
-		dropDmlTriggersNotInSource: boolean;
-		dropExtendedPropertiesNotInSource: boolean;
-		dropIndexesNotInSource: boolean;
-		ignoreFileAndLogFilePath: boolean;
-		ignoreExtendedProperties: boolean;
-		ignoreDmlTriggerState: boolean;
-		ignoreDmlTriggerOrder: boolean;
-		ignoreDefaultSchema: boolean;
-		ignoreDdlTriggerState: boolean;
-		ignoreDdlTriggerOrder: boolean;
-		ignoreCryptographicProviderFilePath: boolean;
-		verifyDeployment: boolean;
-		ignoreComments: boolean;
-		ignoreColumnCollation: boolean;
-		ignoreAuthorizer: boolean;
-		ignoreAnsiNulls: boolean;
-		generateSmartDefaults: boolean;
-		dropStatisticsNotInSource: boolean;
-		dropRoleMembersNotInSource: boolean;
-		dropPermissionsNotInSource: boolean;
-		dropObjectsNotInSource: boolean;
-		ignoreColumnOrder: boolean;
-		doNotDropObjectTypes: SchemaObjectType[];
-		excludeObjectTypes: SchemaObjectType[];
+		ignoreTableOptions: DacDeployOptionPropertyBoolean;
```

Comment:
I am assuming we could not think of a way to avoid needing these defined on ADS side and just use these from STS? @ssreerama  @kisantia

Reply:
Are you asking if we discussed to move these to STS right? I'll look into it and see if I can move all these from ADS to STS itself.